tensorflow/lite/micro/kernels/ceva_bx1/CEVA_basic_op.h

```diff
@@ -0,0 +1,611 @@
+/*************************************************************************************\
+* Copyright (C) CEVA(R) Inc. All rights reserved *

```

Comment:
We can only accept code with a single copyright for the tensorflow authors + apache license.